x-pack/plugins/observability_solution/apm/public/components/shared/links/apm/service_link/index.tsx

```diff
@@ -46,9 +46,9 @@ export function ServiceLink({
 
   const serviceLink = isMobileAgentName(agentName)
     ? '/mobile-services/{serviceName}/overview'
-    : isLogsSignal(signalTypes)
-    ? '/logs-services/{serviceName}/overview'
-    : '/services/{serviceName}/overview';
+    : isApmSignal(signalTypes)
```

Comment:
Q: What is wrong with `isLogsSignal`?

Reply:
Nothing is wrong, but if an entity has APM we should open the APM view it doesn't matter if it also has logs.